### DISCUSSION TITLE
LXD + Caprover

### DISCUSSION THREAD

**Discussion by DanielBanobreDopico** [caprover/caprover#942] [General]
Hi!
I'm in a little developers comunity. We are thinking about hire a server for sharing, using LXD for isolating and limiting resources comsumption for each of us.

Do you know if it's possible to run Caprover into LXD container?

If it will be possible each of us could easily manage his own projects in his own LXD instance.

Thanks!.

**Comment by githubsaturn** [comment]
CapRover apps run in Docker containers, which similar to raw LXD containers, can be manually modified to limit resource usage. But CapRover is not meant for multi tenancy - it doesn't have support for multiple users. You'd have to share the same user.

**Reply by DanielBanobreDopico** [reply]
Yes. I know CapRover uses Docker. But it looks like [Docker can run into LXD containers](https://lxd.readthedocs.io/en/latest/#how-can-i-run-docker-inside-a-lxd-container).

In this way, it look like we can use LXD for user resources isolation, and Docker-CapRover for app management.

What do you think?.

**Reply by githubsaturn** [reply]
Theoretically it may work. But this is surely not what CapRover is designed for. So give it a shot and if it works, great!

**Reply by capriciousduck** [reply]
@DanielBanobreDopico Did you get it working?
